lesson4/decision.md

```diff
@@ -36,11 +36,11 @@ public class TallestPerson {
 }
 ```
 
-Let's break this code fragment apart:
+Let's look into this code fragment in more detail:
 
-### The if-keyword
+### The if keyword
 
-If a block of code shall only be executed if a condition is `true` we have to use the `if`-statement.
+If a block of code should only be executed if a condition is `true` we have to use the `if` statement.
```

Comment:
Small note :) I think we're missing a comma here: "If a block of code should only be executed if a condition is `true`**,** we have to use the `if` statement."